jbcli/jbcli/cli/jb.py

```diff
@@ -553,6 +553,90 @@ def prompt_interval():
     stash.put("interval", answer)
 
 
+@cli.command()
+def add_users():
+    _add_users()
+
+
+def _add_users():
+    if stash.get("users") is not None:
+        return
```

Comment:
I may have messed up my github suggestion but this check should not exist here, only above in the `jb start` code